engine/src/test/java/io/camunda/zeebe/engine/processing/bpmn/activity/BusinessRuleTaskTest.java

```diff
@@ -205,4 +209,129 @@ public void shouldUseResultInOutputMappings() {
         .extracting(VariableRecordValue::getScopeKey, VariableRecordValue::getValue)
         .containsExactly(processInstanceKey, "\"Jedi\"");
   }
+
+  @Test
+  public void shouldWriteDecisionEvaluationEvent() {
+    // given
+    final var deployment =
+        ENGINE
+            .deployment()
+            .withXmlClasspathResource(DMN_RESOURCE)
+            .withXmlResource(
+                processWithBusinessRuleTask(
+                    t ->
+                        t.zeebeCalledDecisionId("force_user").zeebeResultVariable(RESULT_VARIABLE)))
+            .deploy();
+
+    final var calledDecision =
+        deployment.getValue().getDecisionsMetadata().stream()
+            .filter(decision -> decision.getDecisionId().equals("force_user"))
+            .findFirst()
+            .orElseThrow();
```

Comment:
👍 Often we do a `getFirst()` I think, but I like this better. It's nice and explicit